2.1/build/build-rpms.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Download cc-oci-runtime.tar.gz
+curl -OkL https://github.com/01org/cc-oci-runtime/archive/2.1.0-rc.6.tar.gz
+
+# Download qemu-lite.tar.gz
+curl -OkL https://github.com/01org/qemu-lite/archive/da5004e3ffc6a79df82d1b9d8f8533c4045f193c.tar.gz
+
+# Setup cc-oci-runtime
+rpmdev-setuptree
+cd ~/rpmbuild/SOURCES
+mv ~/2.1.0-rc.6.tar.gz ~/rpmbuild/SOURCES
+
+# Move qemu-lite tar
+mv ~/da5004e3ffc6a79df82d1b9d8f8533c4045f193c.tar.gz ~/rpmbuild/SOURCES
+
+# Update cc-oci-runtime tar name
+mv ~/rpmbuild/SOURCES/2.1.0-rc.6.tar.gz  ~/rpmbuild/SOURCES/cc-oci-runtime-2.1.0.rc.6.tar.gz
+
+# Check os-distribution for build requirements for cc-oci-runtime
+if [ `cat /etc/os-release | grep id | cut -d '=' -f2 | head -1` == "centos" ]; then
```

Comment:
if [ `cat /etc/os-release | grep ID | cut -d '=' -f2 | head -1` == "centos" ]; then

ID is in capitals